Content.Server/Anomaly/Effects/ElectricityAnomalySystem.cs

```diff
@@ -58,4 +56,32 @@ private void OnSupercritical(EntityUid uid, ElectricityAnomalyComponent componen
             _lightning.ShootLightning(uid, ent);
         }
     }
+
+    public override void Update(float frameTime)
```

Comment:
move below the initialize()

Reply:
put it in code conventions or hold a gun to my head but the lord will never be able to stop me from putting update loops at the ends of files.